internal/pkg/cli/task_run.go

```diff
@@ -374,6 +382,42 @@ func (o *runTaskOpts) Validate() error {
 	return nil
 }
 
+func (o *runTaskOpts) confirmSecretsAccess() error {
+
+	if o.executionRole != "" {
+		return nil
+	}
+
+	if o.appName == "" && o.env == "" && !o.acknowledgeSecretsAccess {
+		log.Infoln("Looks like you're requesting ssm:GetParameters to the following SSM parameters:")
+
+		for _, value := range o.secrets {
+			//For Systems Manager Parameter Store parameter you can specify it as name or the ARN if it exists in the same Region as the task you are launching.
+			if !arn.IsARN(value) || strings.Contains(value, ":ssm:") {
+				log.Infoln("* " + value)
+			}
+		}
+
+		log.Infoln("\nand secretsmanager:GetSecretValue to the following Secrets Manager secrets:")
+
+		for _, value := range o.secrets {
+			if strings.Contains(value, ":secretsmanager:") {
```

Comment:
Maybe add a comment specifying the reason why we don't need to check if it is an ARN for secret manager. BTW, do we have any validation mechanism for the secret value they pass in via flag?

Reply:
@iamhopaul123  For secret validation mechanism, do you mean like validating if they exist and whether they are tagged?

Reply:
not online check but offline to make sure they have to pass a valid ARN for either ssm or secretmanager

Reply:
otherwise what if they provide a valid ARN but it is not either ssm or secret manager?

Reply:
yeah since it's just gonna be an offline check we can just do it in `Validate()`

Reply:
What is the meaning of online and offline check?

Reply:
offline means we don't need to make API calls that require internet connect.